scout/server/links.py

```diff
@@ -522,7 +522,7 @@ def cbioportal(hgnc_symbol, protein_sequence_name):
 def mutantp53(hgnc_id, protein_variant):
     if hgnc_id != 11998:
         return None
-    if not protein_variant:
+    if not protein_variant or protein_variant.endswith("=") or protein_variant.endswith("%3D"):
```

Comment:
Why does it end like an html-encoded "=" ?

Reply:
Its from VEP, presumably since certain special characters override `"`-escapes/quotes in VCFs. We actually miss that in display in several places that don't use the prettified variant versions..

Reply:
At this time we could actually skip the literal `=` test, but I dunno, better safe than sorry?

Reply:
Yes I agree